### PR TITLE
ADD: Comment image upload system test

### PR DESCRIPTION
**Pull request by VladimirMikulic**
Part of #5316 

@jywarren could you review this? Thanks.

### CODE REVIEW

File: test/system/comment_test.rb [hunk 1]
```diff
@@ -74,4 +74,40 @@ def setup
     assert_equal( comment_preview_button.text, "Preview" )
   end
 
+  test 'comment image upload' do
```
Comment (SidharthBansal) [inline test/system/comment_test.rb]: There should be a test for exceeding image size too

**Comment by SidharthBansal** [comment]
@Uzay-G @VladimirMikulic Please create a pr for Comments.
In the comments pr you can add all the system tests regarding comments.
For example, image upload tests, Cntrl+Enter should result in posting the comment and so on.
It will result in pr floods if you will add one pr per system test. I hope this makes sense.
However if you think that you require functional changes with the system tests, then you can create a discrete pr.

**Comment by SidharthBansal** [comment]
The prs requiring the functional changes will be graded separately. We will be fair to you all in terms of marks.

File: test/system/comment_test.rb (hunk 1 above)
Comment (VladimirMikulic) [reply test/system/comment_test.rb]: The question is, do you have a limit set? I've uploaded 60mb image without a problem.

Comment (VladimirMikulic) [reply test/system/comment_test.rb]: If you don't display some kind of error message or something, then I can't test it.

Comment (SidharthBansal) [reply test/system/comment_test.rb]: Kindly check. We need to set the limit if not set yet. pinging @jywarren for limit

**Comment by Uzay-G** [comment]
> @Uzay-G @VladimirMikulic Please create a pr for Comments.
> In the comments pr you can add all the system tests regarding comments.
> For example, image upload tests, Cntrl+Enter should result in posting the comment and so on.
> It will result in pr floods if you will add one pr per system test. I hope this makes sense.
> However if you think that you require functional changes with the system tests, then you can create a discrete pr.


My comment test has already been merged but we will do this if there are more comments tests :+1:

**Comment by VladimirMikulic** [comment]
@jywarren I think you'll be glad to see this. Both image upload tests have been repaired and are working (even drag & drop)! The system tests are awesome :smile: Love them!

**Comment by jywarren** [comment]
This is EPIC! So great! I'm so glad you like system testing. It's so powerful. Thank you a ton for this! I'll close my original attempt!

**Comment by SidharthBansal** [comment]
Yeah we should do more tests.

**Pull request merged**